plugins/analytics/public/malshare.py

```diff
@@ -35,7 +35,7 @@ def fetch(observable, api_key):
                 'api_key': api_key,
                 'action': 'details'
             }
-            response = requests.get('https://malshare.com/api.php', params)
```

Comment:
I think we should remove `verify=False` here, and let the feed error out in the UI if the certificate is not valid.